functions/go/search-replace/api/pathparser.go

```diff
@@ -1,4 +1,4 @@
-package main
+package api
```

Comment:
If we don't really intend to let our users to consume this package directly, we should avoid using `api` as package name. Probably call it `searchreplace` or something similar.

Reply:
Updated.